anghammarad/src/main/scala/com/gu/anghammarad/messages/Messages.scala

```diff
@@ -36,12 +36,28 @@ object Messages {
   }
 
   def emailMessage(notification: Notification): EmailMessage = {
-    val md = mdParser.parse(notification.message)
+    val actionPrefix =
+      s"""
+         |---------------------
+         |
+       """.stripMargin
+
+    val htmlActions = notification.actions.map { action =>
+      s"[${action.cta}](${action.url})"
+    }.mkString("\n\n")
+    val plainTextActions = notification.actions.map { action =>
+      s"${action.cta} - ${action.url}"
+    }.mkString("\n\n")
+
+    val finalMarkdown = notification.message + actionPrefix + htmlActions
+    val finalPlanText = notification.message + actionPrefix + plainTextActions
+
+    val md = mdParser.parse(finalMarkdown)
     val html = emailContents(md)
 
     EmailMessage(
       notification.subject,
-      notification.message,
+      finalPlanText,
```

Comment:
Minor: is this intended to be `finalPlainText`?

Reply:
oops